PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/models/openrtb/bidRequests/MobileSdkPassThrough.java

```diff
@@ -163,8 +168,19 @@ private MobileSdkPassThrough(JSONObject passThrough) {
                 getAndSave("skipbuttonposition", String.class, it -> skipButtonPosition = Position.fromString(it));
             }
         } catch (JSONException exception) {
-            LogUtil.error(TAG, "Can't parse configuration");
+            LogUtil.error(TAG, "Can't parse adconfiguration");
+        }
+        try {
+            if (passThrough.has("sdkconfiguration")) {
+                configuration = passThrough.getJSONObject("sdkconfiguration");
+                getAndSave("cftbanner", Integer.class, it -> bannerTimeout = it);
+                getAndSave("cftprerender", Integer.class, it -> preRenderTimeout = it);
```

Comment:
If a user set up only one configuration parameter this line will throw an exception. We should support cases with only one parameter.